packages/ceramic-doctype-three-id/src/three-id-doctype.ts

```diff
@@ -100,39 +100,24 @@ export class ThreeIdDoctype extends Doctype {
             header.schema = newSchema
         }
         const record: any = { header, data: patch, prev: doctype.head, id: doctype.state.log[0] }
-        return ThreeIdDoctype._signRecord(record, did)
+        return ThreeIdDoctype._signDagJWS(record, did)
     }
 
     /**
-     * Sign ThreeId record
+     * Sign 3ID record
      * @param did - DID instance
      * @param record - Record to be signed
      * @private
      */
-    static async _signRecord(record: any, did: DID): Promise<any> {
+    static async _signDagJWS(record: any, did: DID): Promise<any> {
         if (did == null || !did.authenticated) {
-            throw new Error('No DID authenticated')
-        }
-        // TODO - use the dag-jose library for properly encoded signed records
-        // convert CID to string for signing
-        const tmpCID = record.prev
-        const tmpId = record.id
-        if (tmpCID) {
-            record.prev = { '/': tmpCID.toString() }
-        }
-        if (tmpId) {
-            record.id = { '/': tmpId.toString() }
+            throw new Error('No user authenticated')
```

Comment:
Can we keep it to `DID` instead of `user` or rename everywhere for consistency please?

Reply:
👍